randovania/generator/filler/pickup_list.py

```diff
@@ -46,7 +46,9 @@ def _unsatisfied_item_requirements_in_list(
     damage = []
 
     for individual in alternative.values():
-        if individual.negate:
+        # We should only skip negative requirements, if we can not satisfy them anymore, as that means
+        # we have lost the chance to do so
+        if individual.negate and not individual.satisfied(state.resoures, state.energy, state.resource_database):
```

Comment:
This looks not correct. `resoures` is a typo and maybe your result will be completly different if you use `resources`

Reply:
Whoops, no idea how that made it in.

Reply:
On a second thought: This is python. Shouldn't it throw an `AttributeError` if this was ever called?